src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/util/ClientUtils.java

```diff
@@ -52,9 +52,9 @@ public static boolean hasValidLocalSignCert(ClientId clientId, List<CertificateI
             String ocspResponseStatus = null;
             try {
                 ocspResponseStatus = OcspUtils.getOcspResponseStatus(certificateInfo.getOcspBytes());
```

Comment:
A test might be justified, since there is a bit of complexity in this method (and the first implementation got it wrong) - maybe a test that would have failed before the fix, and passes after the fix?

Reply:
Added a test